keymaps/julia-client.cson.cmd

```diff
@@ -36,6 +34,19 @@ Any global commands should either be non-default or, ideally, prefixed with `C-J
   'cmd-j cmd-m': 'julia-client:set-working-module'
   'cmd-j cmd-f': 'julia-client:format-code'
 
+# Weave atom-text-editor
+'.platform-darwin atom-text-editor[data-grammar="source weave md"],
+ .platform-darwin atom-text-editor[data-grammar="source weave latex"]':
+  'cmd-enter': 'julia-client:run-block'
+  'shift-enter': 'julia-client:run-and-move'
+  'cmd-shift-enter': 'julia-client:run-all'
+  'alt-enter': 'julia-client:run-cell'
+  'alt-shift-enter': 'julia-client:run-cell-and-move'
+  'alt-down': 'julia-client:next-cell'
+  'alt-up': 'julia-client:prev-cell'
+  'cmd-j cmd-g': 'julia-client:goto-symbol'
+  'cmd-j cmd-d': 'julia-client:show-documentation'
```

Comment:
I feel like this belongs in language-weave -- what's the reason for moving it here?

Reply:
Umm because julia-client defines those commands ?

I mean, when we change the name of the command or something, and accordingly change `keymaps/` or `menus/`, it would be easier to change that for weave files, if we have them here.
(Like avoiding the case like that the context-menu for julia-client is now "Juno" but language-weave one is still "Julia-Client")

Reply:
Hm. language-weave depends on julia-client, so they should make sure to keep up with upstream (and yes, that means I should've made that change there as well :P).

IMHO we should keep those packages as separate as possible.

Reply:
> language-weave depends on julia-client, so they should make sure to keep up with upstream

Sounds fair. Then, let's go with the current direction :)

Do you mind allowing the write access to language-weave for members in "Commiters" team ?
Then I could make such a tiny changes easily.

Reply:
Alright, you should have write access now.

Reply:
confirmed, thx ;)